assets/translations/fr.json

```diff
@@ -86,7 +86,7 @@
   },
   "applet_list_component": {
     "about_title": "À propos de MindLogger",
-    "no_applets_yet": "Aucune applet spécifiée pour le moment",
+    "no_applets_yet": "Vous n'avez aucune applet",
```

Comment:
Vous n’avez pas d’applets
https://www.reverso.net/text-translation#sl=eng&tl=fra&text=You%2520have%2520no%2520applets